Indexing anvil9 snapshot `ANVIL_CMG_Yale_GRU` fails due to joiner

```
[ERROR] AssertionError: R("The string '||' is disallowed in metadata", TDRAnvilBundleFQID(uuid='d2242135-0917-a279-b0c9-4aa227162037', version='2022-06-01T00:00:00.000000Z', source=TDRSourceRef(id='05a9e369-0011-48d9-ab2e-af334973bdb5', spec=TDRSourceSpec(prefix=Prefix(common='', partition=0), type=<Type.bigquery: 'bigquery'>, domain=<Domain.gcp: 'gcp'>, subdomain='datarepo-3f73ca7a', name='ANVIL_CMG_Yale_GRU_20221020_ANV5_202502070356')), table_name='subject', batch_prefix='f'))
Traceback (most recent call last):
  File "/var/task/azul/chalice.py", line 190, in patched_event_source_handler
    return old_handler(self_, event, context)
  File "/var/task/chalice/app.py", line 1761, in __call__
    return self.handler(event_obj)
  File "/var/task/app.py", line 241, in contribute_retry
    app.index_controller.contribute(event, retry=True)
  File "/var/task/azul/indexer/index_controller.py", line 175, in contribute
    contributions, replicas = self.transform(catalog, notification, delete)
  File "/var/task/azul/indexer/index_controller.py", line 226, in transform
    results = service.transform(catalog, bundle, partition, delete=delete)
  File "/var/task/azul/indexer/index_service.py", line 287, in transform
    bundle.reject_joiner()
  File "/var/task/azul/plugins/metadata/anvil/bundle.py", line 121, in reject_joiner
    self._reject_joiner(chain(self.entities.values(), self.orphans.values()))
  File "/var/task/azul/indexer/__init__.py", line 608, in _reject_joiner
    assert joiner not in chunk, R(
```

`||` found in three rows of the `datarepo-3f73ca7a.ANVIL_CMG_Yale_GRU_20221020_ANV5_202502070356.subject` table:

```
[
  {
    "catalog": "anvil9",
    "table": "datarepo-3f73ca7a.ANVIL_CMG_Yale_GRU_20221020_ANV5_202502070356.subject",
    "column": "disease_id",
    "row_id": "02f72858-6660-4fc0-a2bc-b899a8e2eef3",
    "value": "OMIM:613229||OMIM:607834"
  },
  {
    "catalog": "anvil9",
    "table": "datarepo-3f73ca7a.ANVIL_CMG_Yale_GRU_20221020_ANV5_202502070356.subject",
    "column": "disease_id",
    "row_id": "fbd90528-9e96-4471-99db-99ca13915948",
    "value": "OMIM:613229||OMIM:607834"
  },
  {
    "catalog": "anvil9",
    "table": "datarepo-3f73ca7a.ANVIL_CMG_Yale_GRU_20221020_ANV5_202502070356.subject",
    "column": "disease_id",
    "row_id": "9e747933-ca57-4fe5-8f56-fdeedf5587dc",
    "value": "OMIM:613229||OMIM:164230|OMIM:607834"
  }
]
```

--- a/deployments/anvilprod/environment.py
+++ b/deployments/anvilprod/environment.py
@@ -933,7 +933,7 @@ anvil8_sources = mkdict(anvil7_sources, 254, mkdelta([
     # @formatter:on
 ]))
 
-anvil9_sources = mkdict(anvil8_sources, 285, mkdelta([
+anvil9_sources = mkdict(anvil8_sources, 284, mkdelta([
     # @formatter:off
     mksrc('bigquery', 'datarepo-b555b2f5', 'ANVIL_1000G_PRIMED_data_model_20240410_ANV5_202502211647'),
     mksrc('bigquery', 'datarepo-265cde27', 'ANVIL_ALSCompute_Collection_GRU_20231016_ANV5_202410310107'),
@@ -968,7 +968,6 @@ anvil9_sources = mkdict(anvil8_sources, 285, mkdelta([
     mksrc('bigquery', 'datarepo-0e0af0a8', 'ANVIL_CMG_UWash_DS_EP_20250219_ANV5_202502201854'),
     mksrc('bigquery', 'datarepo-766c47dd', 'ANVIL_CMG_UWash_GRU_20250224_ANV5_202502241706'),
     mksrc('bigquery', 'datarepo-aa22e87e', 'ANVIL_CMG_UWash_GRU_IRB_20250224_ANV5_202502241723'),
-    mksrc('bigquery', 'datarepo-3f73ca7a', 'ANVIL_CMG_Yale_GRU_20221020_ANV5_202502070356'),
     mksrc('bigquery', 'datarepo-7c44dbc8', 'ANVIL_CMH_GAFK_ES_20240301_ANV5_202502201925'),
     mksrc('bigquery', 'datarepo-97f2fa00', 'ANVIL_CMH_GAFK_GS_long_read_20240301_ANV5_202502201932'),
     mksrc('bigquery', 'datarepo-456a8996', 'ANVIL_CMH_GAFK_IlluminaGSA_20240311_ANV5_202502201937'),

--- a/deployments/hammerbox/environment.py
+++ b/deployments/hammerbox/environment.py
@@ -935,7 +935,7 @@ anvil8_sources = mkdict(anvil7_sources, 254, mkdelta([
     # @formatter:on
 ]))
 
-anvil9_sources = mkdict(anvil8_sources, 285, mkdelta([
+anvil9_sources = mkdict(anvil8_sources, 284, mkdelta([
     # @formatter:off
     mksrc('bigquery', 'datarepo-b555b2f5', 'ANVIL_1000G_PRIMED_data_model_20240410_ANV5_202502211647'),
     mksrc('bigquery', 'datarepo-265cde27', 'ANVIL_ALSCompute_Collection_GRU_20231016_ANV5_202410310107'),
@@ -970,7 +970,6 @@ anvil9_sources = mkdict(anvil8_sources, 285, mkdelta([
     mksrc('bigquery', 'datarepo-0e0af0a8', 'ANVIL_CMG_UWash_DS_EP_20250219_ANV5_202502201854'),
     mksrc('bigquery', 'datarepo-766c47dd', 'ANVIL_CMG_UWash_GRU_20250224_ANV5_202502241706'),
     mksrc('bigquery', 'datarepo-aa22e87e', 'ANVIL_CMG_UWash_GRU_IRB_20250224_ANV5_202502241723'),
-    mksrc('bigquery', 'datarepo-3f73ca7a', 'ANVIL_CMG_Yale_GRU_20221020_ANV5_202502070356'),
     mksrc('bigquery', 'datarepo-7c44dbc8', 'ANVIL_CMH_GAFK_ES_20240301_ANV5_202502201925'),
     mksrc('bigquery', 'datarepo-97f2fa00', 'ANVIL_CMH_GAFK_GS_long_read_20240301_ANV5_202502201932'),
     mksrc('bigquery', 'datarepo-456a8996', 'ANVIL_CMH_GAFK_IlluminaGSA_20240311_ANV5_202502201937'),

